intro-pkgs.Rmd

```diff
@@ -21,6 +21,31 @@ What are the challenges of HTTP testing?
 * Having tests depend on having secrets for authentication at hand is not ideal.
 * Having tests for situations that are hard to trigger (e.g. the failure of a remote server) is tricky.
 
+## Tradeoffs
+
+The below packages help you meet these challenges by faking web servers or mocking their responses.
+Both approaches let you run your tests against a state of an external HTTP API which you fully control.
+This can be a good thing, because it allows you to test against all theoretically possible behavior of the API, as opposed to just the current behavior of the live API.
+
+More fundamentally, by taking control of the API behavior, your can test your code, and only your code.
```

Comment:
> taking control of the API behavior

can you clarify this?

Reply:
I just got rid of this convoluted language.